Makefile

```diff
@@ -37,6 +37,18 @@ deploy:
 	# give bitcoind rpc server time to start
 	sleep 5
 
+# Public test network
+testuser-desktop:
```

Comment:
The makefile already has stagenet configs for alice, bob, and charlie. Maybe rather than adding a 4th, we reference one of those, and rename them if necessary?